source/_patterns/04-pages/article--research.mustache

```diff
@@ -7,6 +7,14 @@
     </div>
     <main role="main" class="main blg-margin-bottom-large" id="maincontent">
 
+      {{#dismissible}}
+        <div class="wrapper">
+          <div class="content-container-grid">
```

Comment:
The challenge accepting this in its current form is that we need to wrap the dismissible info bar in the `content-container-grid` so that it can be presented as we want but no other info bars work well wrapped in the same elements.

The info bar should be presented as you expected them to  be in the atoms preview, not only when viewed in a page with markup that doesn't work for all cases.

Reply:
The additional wrappers are removed, all styles are changed for "info-bar--dismissible" class